scripts/start/submit.R

```diff
@@ -62,26 +62,15 @@ submit <- function(cfg, restart = FALSE, stopOnFolderCreateError = TRUE) {
 
         if (getOption("autoRenvUpdates", FALSE)) {
           source("scripts/utils/updateRenv.R")
-        } else {
-          packagesUrl <- "https://pik-piam.r-universe.dev/src/contrib/PACKAGES"
-          pikPackages <- sub("^Package: ", "", grep("^Package: ", readLines(packagesUrl), value = TRUE))
-          installed <- utils::installed.packages()
-          outdatedPackages <- utils::old.packages(instPkgs = installed[installed[, "Package"] %in% pikPackages, ])
-          if (!is.null(outdatedPackages)) {
-            message("The following PIK packages can be updated:\n",
-                    paste("-", outdatedPackages[, "Package"], ":",
-                          outdatedPackages[, "Installed"], "->", outdatedPackages[, "ReposVer"],
-                          collapse = "\n"),
-                    "\nConsider updating with `Rscript scripts/utils/updateRenv.R`.")
-          }
+        } else if (!is.null(lucode2::showUpdates())) {
+          message("Consider updating with `Rscript scripts/utils/updateRenv.R`.")
```

Comment:
Should we tell users to use make now in these cases @mikapfl ?
```suggestion
          message("Consider updating with `make update-renv`.")
```

Reply:
For linux users, `make` will be easier. People running on Windows probably want `Rscript`. I'd say, leave it as it is.